### PR TITLE
Updating messages

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

File: src/components/single-form.js [hunk 1]
```diff
@@ -32,8 +32,12 @@ var singleForm = React.createClass({
     };
   },
   renderPrivacyPolicy: function() {
+    var privicyPolicyMessage = "privacy_policy_var_b";
```
Comment (TheoChevalier) [inline src/components/single-form.js]: typo, privicy (and below)

**Comment by ScottDowne** [comment]
@TheoChevalier Good catch, updated.

**Comment by ScottDowne** [comment]
@TheoChevalier any thoughts on l10n for this and timing?

**Comment by ScottDowne** [comment]
@TheoChevalier just in case I put this behind a lang check, mainly because we're landing captcha this week and I was worried about prod potentially being blocked while we sort out any unexpected stuff with that, so I didn't feel comfortable letting in l10n translations on this, depending on how captcha goes, so just to be safe, I put this behind the lang check so we can potentially land it before captcha for at least english.

**Comment by TheoChevalier** [comment]
@ScottDowne Ok, thanks! Feel free to merge whenever it’s ready (no need to coordinate with me here, as I don’t have to send a heads up)

**Pull request merged**